File: Zero Judge Python/a005.py
n = int(input())
for _ in range(n):
    a,b,c,d = map(int,input().split())
    if(b-a == c-b):
        Difference = b - a
        e = int(d+Difference)
        print(f"{a} {b} {c} {d} {e}")
    else:
        Ratio = b/a
        e = int(d*Ratio)
        print(f"{a} {b} {c} {d} {e}")

File: Zero Judge C/a005.c
#include <stdio.h>

int main(){
	int n;
	scanf("%d",&n);
	//getchar();
	for(int i=0 ; i<n ; i++){
		int a,b,c,d;
		scanf("%d %d %d %d",&a,&b,&c,&d);
		if(b-a==c-b){
			int difference = b-a;
			int e = d + difference;
			printf("%d %d %d %d %d\n",a,b,c,d,e);
		}
		else{
			int ratio = b/a;
			int e = d*ratio;
			printf("%d %d %d %d %d\n",a,b,c,d,e);
		}
	}
	return 0;
}